core/tests/puppeteer-acceptance-tests/utilities/user/logged-in-user.ts

```diff
@@ -176,9 +178,23 @@ export class LoggedInUser extends BaseUser {
    * Navigates to the learner dashboard.
    */
   async navigateToLearnerDashboard(): Promise<void> {
+    // Reloading the page to ensure the latest state is reflected.
+    // await this.page.reload();
     await this.goto(learnerDashboardUrl);
   }
 
+  /**
+   * Navigates to the learner dashboard using profile button in the navbar.
+   */
+  async navigateToLearnerDashboardUsingProfileButton(): Promise<void> {
+    await this.clickOn(profileButton);
+    await this.clickOn(learnerDashboardDropdown);
```

Comment:
@seanlip PTAL!! This is also working properly.

Reply:
Yup this approach seems reasonable! Please remove all commented-out code, thanks.

Reply:
Done!